### PR TITLE
The Preview is now the official API (no more view2D)

### PR DESCRIPTION
**Pull request by gjn**
This PR removes the preview of the loader and makes it the official version that will be served. The examples and documentation inside mf-chsdi3 are adapted accordingly (mainly replacing View2D with View). Note however, that old simple code using View2D should still work (we use alias), but might break in other areas.

I've kept the old versions (old-ga.js and old-ga-whitespace.js) around to be able to switch back to the old version quickly. (Year-end and all...)

Once this is deployed, we need to adapt the API documentation hosted on github (-> https://github.com/geoadmin/ol3/pull/112) and also our examples in codepen.$

Note that this also adresses #1006

### CODE REVIEW

**Comment by gjn** [comment]
I've rebased.

@kandre Could you apply the css changes? Maybe best directly to https://github.com/geoadmin/ol3 and I'll update then this PR accordingly.

File: chsdi/templates/lubis_map.mako [hunk 1]
```diff
@@ -36,7 +41,7 @@
                   }
 
                   curInstance = (++curInstance > MAX_INSTANCES) ? 0 : curInstance;
-                  return url.replace('{curInstance}', curInstance) + tileCoord.toString() + ".jpg";
+                  return url.replace('{curInstance}', curInstance) + tileCoordToString(tileCoord) + ".jpg";
```
Comment (oterral) [inline chsdi/templates/lubis_map.mako]: You can use `tileCoord.join(',') ` because it's an array

**Comment by oterral** [comment]
@gjn  I fix the attribution  css in the api --> https://github.com/geoadmin/ol3/pull/114/files

**Comment by gjn** [comment]
@oterral I've adapted after merging your fix. But it does not look correct.

Please see [here](http://mf-chsdi3.dev.bgdi.ch/ltjeg/) or [here](http://mf-chsdi3.dev.bgdi.ch/ltjeg/examples/geoadmin_mapoverlay.html)

**Comment by oterral** [comment]
ök i look into it

**Comment by oterral** [comment]
Hope it's good now: https://github.com/geoadmin/ol3/pull/115/files

**Comment by gjn** [comment]
Had to correct a little bit, but looks good now.

**Comment by cedricmoullet** [comment]
@pauloamado Have you tested the usage of the API3 in the CMS with the preview mode ?

**Comment by gjn** [comment]
@cedricmoullet As discussed yesterday, we want this for today's deploy, right?

**Pull request merged**